src/components/AuditLogsTable/_audit-logs-table.scss

```diff
@@ -0,0 +1,23 @@
+@import "vanilla-framework/scss/vanilla";
+@import "../../scss/breakpoints";
+
+.audit-logs-table {
+  td,
+  th {
+    // Facade version.
+    &:nth-child(6) {
+      width: 10%;
+    }
+
+    @include mobile {
```

Comment:
I've observed that there are no columns being hidden on medium size screens, which was done for the other tables. Should it be like this?

Reply:
Thank, yep in this case the columns didn't need adjusting for the medium size.